api-admin/src/main/java/bio/terra/pearl/api/admin/controller/SiteImageController.java

```diff
@@ -48,6 +54,19 @@ public ResponseEntity<Object> list(String portalShortcode) {
     return ResponseEntity.ok(siteImageExtService.list(portalShortcode, operator));
   }
 
+  @Override
+  public ResponseEntity<Object> upload(
```

Comment:
![image](https://github.com/broadinstitute/juniper/assets/129115982/2462389f-15c6-49c7-aa35-cd1683ffec89)

Can we check for file type?
Is it possible to catch this error and present something better? I realize this is in the bowels of the Web server.